tensorflow_addons/image/testing_imageops.ipynb

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
<p>Could you remove click the "x" on the top left of this output to remove the install logging.</p><p> </p><p>Also please use </p><p><code></code><code></p><p>!pip install tensorflow-gpu==2.0.0-beta1</p><p>!pip install tensorflow-addons</p><p></code><code></code></p><p>TFA should be 0.4</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>